src/pages/admin-create-account/admin-create.jsx

```diff
@@ -0,0 +1,35 @@
+import './admin-create.css';
+import { React, useState } from 'react';
+import CreateAdminConfirmationModal from '../../components/CreateAdminConfirmationModal/CreateAdminConfirmationModal';
```

Comment:
🚫 **[eslint]** <[import/extensions](https://github.com/import-js/eslint-plugin-import/blob/v2.25.2/docs/rules/extensions.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing file extension for "../../components/CreateAdminConfirmationModal/CreateAdminConfirmationModal"